### DISCUSSION TITLE
Is it possible that the plaintext templates are not overwritten or deleted

### DISCUSSION THREAD

**Discussion by a-smily** [maizzle/framework#1382] [Q&A] [answered]
Hello, one more question :)

I create the templates with corresponding plaintext templates.
I then edit the plaintext templates by hand because it is easier than working with `<not-plaintext>` etc.
Is it possible that the plaintext templates are not overwritten or deleted and could remain in the destination path?

With "omit" they are unfortunately always deleted?
Perhaps there is a way to do this without creating an extra directory for the plaintext templates that maizzle does not have access to?

**Comment by cossssmin** [comment, marked as answer]
Unfortunately no, the destination directory is always cleaned up.
